js/languages/ruby/Parser.ml

```diff
@@ -1,5 +1,9 @@
-let parse_pattern _ _ str =
-  let any = Parse_ruby.any_of_string str in
+let parse_pattern print_errors _ str =
+  let any =
+    str
+    |> Pfff_or_tree_sitter.run_pattern ~print_errors
+         [ TreeSitterPat Parse_ruby_tree_sitter.parse_pattern ]
+  in
```

Comment:
It seems that we only use dyp for ruby patterns. I'm not why sure it breaks with  5.0.0 but it seems the tree sitter parse pattern works fine. Is there a reason we're not using it in Parse_pattern/not to use it here?

Reply:
If we're ok with it, this commit fixes the failing JS test.